python/grass/temporal/temporal_granularity.py

```diff
@@ -344,164 +423,83 @@ def compute_absolute_time_granularity(maps):
 
     """
 
-    has_seconds = False
-    has_minutes = False
-    has_hours = False
-    has_days = False
-    has_months = False
-    has_years = False
-
-    use_seconds = False
-    use_minutes = False
-    use_hours = False
-    use_days = False
-    use_months = False
-    use_years = False
-
-    delta = []
-    datetime_delta = []
+    # Create a granularity dict with time units of increasing length
+    # that covers all possible keys in the result of compute_datetime_delta
+    # The order of the keys is important so that loops over the dictionalry
```

Comment:
```suggestion
    # The order of the keys is important so that loops over the dictionary
```

Reply:
Fixed in [04d4778](https://github.com/OSGeo/grass/pull/3350/commits/04d4778d32e34a4c3842187fd22bb7fbc4a8d34e)